configuration/pih/htmlforms/mentalHealthIntake.xml

```diff
@@ -211,9 +211,27 @@
         </style>
     </ifMode>
 
+    <script type="text/javascript">
+        const patientUuid = '<lookup expression="patient.uuid"/>';
+        const encounterUuid = '<lookup expression="encounter.uuid"/>';
+        const encounterDateYmd = '<lookup complexExpression="$fn.formatDate($encounter.encounterDatetime, 'yyyy-MM-dd')"/>';
+        const pihemr = new PihServices(jq);
+        const hfe = new HtmlFormFunctions(pihemr, patientUuid, encounterUuid, encounterDateYmd);
+
+        function toggleConditionSections() {
+            jq('#mh-diagnoses').hide();
+            hfe.showSectionBasedOnCondition('PIH:14650', '#mh-diagnoses', false, true, true);
+        }
+
+        // When the form initially loads, and any time the encounter date is changed, show/hide sections based on condition list
+        jq('#encounterDate').find('input[type="hidden"]').change(function() {
+            toggleConditionSections()
+        });
+        toggleConditionSections();
```

Comment:
do we not need to wrap this in a jq(function())?  Granted, I've totally forgotten a lot of this, but doesn't that do something to make sure it doesn't execute until the DOM is fully mounted or something?

Reply:
Good point, done.